src/routes/faq/sections/dropdown-faqs.svelte

```diff
@@ -0,0 +1,120 @@
+<script lang="ts">
+    import { marked } from "marked";
+    import type { Faqs } from "$lib/types/components";
+
+    export let data: Faqs;
+    export let isExpandable: boolean = false;
+    let activeSubsectionIds = [];
+
+    const handleClick = subsectionId => {
+        if (isExpandable) {
+            let _activeSubsectionIds = activeSubsectionIds;
+            _activeSubsectionIds.includes(subsectionId)
+            ? _activeSubsectionIds.splice(activeSubsectionIds.indexOf(subsectionId), 1)
+            : _activeSubsectionIds.push(subsectionId);
+            activeSubsectionIds = _activeSubsectionIds;
+        }
+    };
+</script>
+
+<div class="w-full">
+    <div id="contents">
+        {#each data?.sections as section, index}
+            <div class={`text-36 leading-110 font-bold mb-10 text-blue-600 ${index > 0 && "mt-20"}`}>
+                {section.title}
+            </div>
+            <div id={section.id} class="border border-grey-100 rounded-2xl bg-white py-5">
+                {#each section?.subsections as subsection}
+                    <div
+                        id={subsection.id}
+                        class={`item first:pb-5 first:pt-0 last:pt-5 last:pb-0 py-5 ${
+                            section.subsections.lastIndexOf(subsection) === section?.subsections.length - 1
+                                ? ""
+                                : "subsection-border relative"
+                        }`}
+                    >
+                        <div
+                            class={`mx-9 my-2 ${isExpandable ? "cursor-pointer" : ""}`}
+                            on:click={() => handleClick(subsection.id)}
```

Comment:
I would split all this into a standalone accordion component, that way you can resuse it if needed and you dont have to store the array of active sections anywhere, but each component will take care of its own thing